### PR TITLE
fix: move encrypted icon to excalidraw-app

### PR DESCRIPTION
**Pull request by ad1992**
An attempt to fix https://github.com/excalidraw/excalidraw/issues/2899

When restoring zenmode the encrypted icon since now is rendered from excal-app via `renderFooter` prop hence gets pushed from right (due to translate) which might look jerky.

### CODE REVIEW

**Comment by dwelle** [comment]
Let's take the opportunity a clean it up a bit:

- make `.App-menu_bottom` `grid-template-columns` into `min-content auto min-content`
- use the center column only for user custom footer; move the language picker + help into the right-most column
- remove all the `position: absolute` from the footer elements

**Comment by ad1992** [comment]
> * use the center column only for user custom footer; move the language picker + help button into the right-most column

The language picker belongs to the custom footer as well, If we split it into 3 cols and use the center column for the custom footer then in zen mode won't it become tricky to handle the animation?

**Comment by dwelle** [comment]
> The language picker belongs to the custom footer as well

Oh, forgot. Good point. 

> If we split it into 3 cols and use the center column for the custom footer then in zen mode won't it become tricky to handle the animation?

Isn't it tricky to handle it even now with respect to the shield icon? It has the same issue. So if we want to solve it from the animation perspective, we can either split the custom footer into `renderFooterLeft` & `renderFooterRight`, or do the animation differently — e.g. animate the center footer column downwards: I think that's solution I'd go wit.

**Comment by ad1992** [comment]
>Isn't it tricky to handle it even now with respect to the shield icon? It has the same issue. So if we want to solve it from the animation perspective, we can either split the custom footer into renderFooterLeft & renderFooterRight, or do the animation differently — e.g. animate the center footer column downwards: I think that's solution I'd go wit.

Yeah separate animation for center column sounds good 👍🏻

**Comment by dwelle** [comment]
Looking :fire:

**Review by dwelle — APPROVED** [review]

File: src/packages/excalidraw/CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ Please add the latest change on the top under the correct section.
 
   This also removes the GitHub icon, keeping it local to the https://excalidraw.com app.
 
+### Fixes
+
+- The encryption shield icon is now removed from excalidraw package as it was specific to excalidraw app and rendered via [`renderFooter`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#renderFooter) prop. In case you were hiding this icon earlier, you need not do that anymore [#3577](https://github.com/excalidraw/excalidraw/pull/3577).
```
Comment (dwelle) [inline src/packages/excalidraw/CHANGELOG.md]: Was it rendered via `renderFooter`? I think it wasn't, which was the point of this PR?

Comment (ad1992) [reply src/packages/excalidraw/CHANGELOG.md]: Yes, I will better update it to `and now rendered via `renderFooter` prop to avoid the confusion

Comment (ad1992) [reply src/packages/excalidraw/CHANGELOG.md]: Actually it was already updated, you can check https://github.com/excalidraw/excalidraw/pull/3577/commits/b6e7a9f6dc85f647eaa6180b656275369db990ae

Comment (dwelle) [reply src/packages/excalidraw/CHANGELOG.md]: oh, I guess GitHub showed me an old commit, sorry!

Comment (dwelle) [reply src/packages/excalidraw/CHANGELOG.md]: Merge whenever you're ready!

Comment (dwelle) [reply src/packages/excalidraw/CHANGELOG.md]: Actually, haven't reviewed readme yet. Gimme a min!

Comment (dwelle) [reply src/packages/excalidraw/CHANGELOG.md]: Done!

Comment (ad1992) [reply src/packages/excalidraw/CHANGELOG.md]: Thank you 👍🏻

**Pull request merged**